tests/network/expose.go

```diff
@@ -121,16 +166,19 @@ var _ = SIGDescribe("[rfe_id:253][crit:medium][vendor:cnv-qe@redhat.com][level:c
 				By("Getting back the cluster IP given for the service")
 				svc, err := virtClient.CoreV1().Services(tcpVM.Namespace).Get(context.Background(), serviceName, k8smetav1.GetOptions{})
 				Expect(err).ToNot(HaveOccurred())
-				serviceIP := svc.Spec.ClusterIP
+				validateClusterIp(svc.Spec.ClusterIP, ipFamily)
 
-				By("Starting a job which tries to reach the VMI via ClusterIP")
-				job := runHelloWorldJob(serviceIP, servicePort, tcpVM.Namespace)
-
-				By("Waiting for the job to report a successful connection attempt")
-				Expect(tests.WaitForJobToSucceed(job, 420*time.Second)).To(Succeed())
+				By("Iterating over the ClusterIPs")
+				serviceIPs := svc.Spec.ClusterIPs
+				for i, ip := range serviceIPs {
```

Comment:
Reading this again and again is still hard to understand. The condition on the `i` is killing this.

Reply:
Would it be better if I change `i` to `ipNumber`?

Reply:
I do not think so...
Why do you think this is worse?
```
createAndWaitForJobToSucceed(runHelloWorldJob, tcpVM.Namespace, serviceIPs[0], servicePort, fmt.Sprintf("%dst ClusterIP", serviceIPs[0]), 420)
assert.XFail(xfailError, func() {
	createAndWaitForJobToSucceed(runHelloWorldJob, tcpVM.Namespace, serviceIPs[1], servicePort, fmt.Sprintf("%dst ClusterIP", serviceIPs[1]), 420)})
```